rootfs/usr/local/etc/php/php.ini

```diff
@@ -0,0 +1,63 @@
+[PHP]
+engine = On
+short_open_tag = Off
+asp_tags = Off
+precision = 14
+output_buffering = 4096
+zlib.output_compression = Off
+implicit_flush = Off
+unserialize_callback_func =
+serialize_precision = 17
+disable_functions =
+disable_classes =
+zend.enable_gc = On
+expose_php = On
+max_execution_time = 30
+max_input_time = 60
+memory_limit = 256M
+error_reporting = Off
+display_errors = Off
+display_startup_errors = Off
+log_errors = On
+log_errors_max_len = 1024
+ignore_repeated_errors = Off
+ignore_repeated_source = Off
+report_memleaks = On
+track_errors = On
+html_errors = On
+variables_order = "GPCS"
+request_order = "GP"
+register_argc_argv = Off
+auto_globals_jit = On
+post_max_size = 200M
+auto_prepend_file =
+auto_append_file =
+default_charset = "UTF-8"
+enable_dl = Off
+file_uploads = Off
+allow_url_fopen = On
+allow_url_include = Off
+default_socket_timeout = 600
+[CLI Server]
+cli_server.color = On
+[Date]
+date.timezone = 'Europe/Madrid'
+[OCI8]
+[Sybase-CT]
+sybct.allow_persistent = On
+sybct.max_persistent = -1
+sybct.max_links = -1
+sybct.min_server_severity = 10
+sybct.min_client_severity = 10
+[bcmath]
+bcmath.scale = 0
+[Tidy]
+tidy.clean_output = Off
+[soap]
+soap.wsdl_cache_enabled=1
+soap.wsdl_cache_dir="/tmp"
+soap.wsdl_cache_ttl=86400
+soap.wsdl_cache_limit = 5
+[ldap]
+ldap.max_links = -1
+error_log = /var/log/php-error_log
```

Comment:
I think it's better to log the errors to STDERR instead

Reply:
Yes. I'll modify it!